cartography/cli.py

```diff
@@ -126,6 +126,45 @@ def _build_parser(self):
                 'respects the AWS CLI/SDK environment variables and does not override them.'
             ),
         )
+        parser.add_argument(
+            '--azure-sync-all-subscriptions',
+            action='store_true',
+            help=(
+                'Enable Azure sync for all discovered profiles. When this parameter is supplied cartography will '
```

Comment:
Do you mean "all discovered subscriptions" instead of "all discovered profiles"?

Reply:
fixed in latest version